client/src/components/molecules/AgendaCard/PreparingAgendaCard.tsx

```diff
@@ -15,22 +17,18 @@ interface Props {
 
 export const PreparingAgendaCard: React.FC<Props> = ({ agenda }) => (
   <Card>
-    <Box gap={8}>
+    <div css={[gap(8)]}>
```

Comment:
하나만 쓸 때에는 `[]`를 쓰지 않아도 돼요!
```suggestion
    <div css={gap(8)}>
```